PhysicsTools/PatAlgos/plugins/CaloJetSlimmer.cc

```diff
@@ -0,0 +1,63 @@
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "DataFormats/Common/interface/RefVector.h"
+
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/ObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleElementCollectionSelector.h"
+
+#include "DataFormats/JetReco/interface/CaloJet.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
```

Comment:
I think that it is better to hold this code here in this package, in view of possible evolution and also the scope of the workflows (miniAOD only).